packages/types/src/events/steps-from-apps.ts

```diff
@@ -0,0 +1,86 @@
+export interface WorkflowDeletedEvent {
```

Comment:
Should we rename this file `workflow`? These are going away, anyway, so maybe not important at all.

Reply:
Yeah I muddled over that - do you think that's better? If so I'll rename it! Indeed, presumably next month when SFA is 🔪 and hopefully I get to a types v3 by then, this will all go away.

Reply:
No need to change it if it's getting tossed, anyway, IMO